src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/RequestHandler.java

```diff
@@ -119,26 +121,26 @@ public static void assignRequest(final IStandardRequestManager manager, final IR
 
         request.setState(new WrappedStaticStateRequestManager(manager), RequestState.ASSIGNING);
 
-        final Set<TypeToken> requestTypes = ReflectionUtils.getSuperClasses(request.getRequestType());
+        @SuppressWarnings(RAWTYPES) final Set<TypeToken> requestTypes = ReflectionUtils.getSuperClasses(request.getRequestType());
         requestTypes.remove(TypeConstants.OBJECT);
 
         final Set<IToken<?>> failedResolvers = new HashSet<>();
 
-        for (final TypeToken requestType : requestTypes)
+        for (@SuppressWarnings(RAWTYPES) final TypeToken requestType : requestTypes)
         {
             if (!manager.getRequestClassResolverMap().containsKey(requestType))
             {
                 continue;
             }
 
-            Collection<IRequestResolver> resolversForRequestType = manager.getRequestClassResolverMap().get(requestType);
+            Collection<IRequestResolver<?>> resolversForRequestType = manager.getRequestClassResolverMap().get(requestType);
             resolversForRequestType = resolversForRequestType.stream()
                                         .filter(r -> r.getRequestType().isAssignableFrom(request.getRequestType()))
                                         .filter(r -> !failedResolvers.contains(r.getRequesterId()))
                                         .sorted(Comparator.comparing(r -> -1 * r.getPriority()))
                                         .collect(Collectors.toSet());
 
-            for (final IRequestResolver resolver : resolversForRequestType)
+            for (final IRequestResolver<?> resolver : resolversForRequestType)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS135)